internal/pkg/api/tinkerbell.go

```diff
@@ -1,96 +1,56 @@
 package api
 
 import (
-	"fmt"
 	"os"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"sigs.k8s.io/yaml"
 
 	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/cluster"
 	"github.com/aws/eks-anywhere/pkg/providers"
-	"github.com/aws/eks-anywhere/pkg/templater"
 )
 
 type TinkerbellConfig struct {
-	clusterConfig    *anywherev1.Cluster
```

Comment:
why we have this config change on tinkerbell?

Reply:
This field wasn't needed and removing it simplified the code